docs/usage/datamon_config_set.md

```diff
@@ -0,0 +1,59 @@
+**Version: dev**
```

Comment:
the md file `datamon_config_create.md`  should be deleted. The git hook does not take care of that automatically yet. Maybe it is the right time to add this (or I'll do it later).

Reply:
thanks!

Reply:
done